src/sysinfo/index.ts

```diff
@@ -36,6 +36,13 @@ export async function getLoadavg() {
   } catch (err) {}
 }
 
+export async function getInternetState() {
+  try {
+    const res = await si.inetChecksite('https://1.1.1.1/');
```

Comment:
This url should be obtained through the configuration and can be set to the configured default value